contrib/externs/showdown-1.3.js

```diff
@@ -0,0 +1,37 @@
+/*
+ * Copyright 2015 The Closure Compiler Authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+// This file intentionally left blank. It's only used for testing externs
+// compilation.
+// This is not an @externs file but the annotation_test requires this line.
+
+
+/**
+ * @const
+ */
+var showdown = {};
+
+/**
+ * @param {*} opts
```

Comment:
Looks like `opts` is at least an optional `Object`? In that case, it should be renamed to `opt_options` and typed as: `@param {Object=} opt_options`.